guides/common/modules/proc_managing-packages-on-the-base-operating-system.adoc

```diff
@@ -10,50 +10,30 @@ The `{foreman-maintain} packages` command restarts some services on the operatin
 ====
 
 .Procedure
-* To install packages on {ProjectServer} or {SmartProxyServer}, enter the following command:
+* To install packages on {ProjectServer} or {SmartProxyServer}:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # {project-package-install} _package_1_ _package_2_
 ----
-* To update specific packages on {ProjectServer} or {SmartProxyServer}, enter the following command:
+* To check the packages update on {ProjectServer} or {SmartProxyServer}:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
-# {project-package-update} _package_1_ _package_2_
+# {project-package-check-update}
 ----
-* To update all packages on {ProjectServer} or {SmartProxyServer}, enter the following command:
+* To update specific packages on {ProjectServer} or {SmartProxyServer}:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
-# {project-package-update}
+# {project-package-update} _package_1_ _package_2_
 ----
-
-.Using yum to Check for Package Updates
-If you want to check for updates using `yum`, enter the command to install and update packages manually and then you can use `yum` to check for updates:
-
-[options="nowrap" subs="+quotes,attributes"]
+* To update all packages on {ProjectServer} or {SmartProxyServer}:
```

Comment:
In my mind it's more logical first recommend updating all packages and only then specific packages, but I'll let other writers weigh in on that.

Reply:
I agree with @ekohl.

Reply:
Addressed :+1: